tensorboard/data/provider.py

```diff
@@ -248,6 +248,42 @@ def read_scalars(
         """
         pass
 
+    @abc.abstractmethod
+    def read_last_scalars(
+        self,
+        ctx=None,
+        *,
+        experiment_id,
+        plugin_name,
+        run_tag_filter=None,
+    ):
+        """Read the most recent values from scalar time series.
+
+        Note that some impementations might not support not all scalar types.
```

Comment:
Grammar nit: might not support not all? Can you rephrase it? Also, did you mean to say not all "plugin" types?

Reply:
Oops, that's a typo, thanks for catching this! Also I I forgot scalars are all numeric (https://www.tensorflow.org/api_docs/python/tf/summary/scalar), I think I was mixing the concept of scalar with tensor. In this case we don't need this note at all, removed.